src/styles/richtext.css

```
@@ -21,12 +21,12 @@
 
 .ql-toolbar button.ql-active .ql-stroke {
   fill: none;
-  stroke: orange !important;
+  stroke: #2D89C8 !important;
```

```suggestion
  @apply !stroke-blue-d1
```
removes the need to repeat the color declaration